helm/hocs-queue-tool/templates/_helpers.tpl

```diff
@@ -0,0 +1,26 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "hocs-app.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
```

Comment:
Should this be 53 characters, see tip here: https://helm.sh/docs/chart_template_guide/getting_started/#adding-a-simple-template-call

> TIP: The name: field is limited to 63 characters because of limitations to the DNS system. For that reason, release names are limited to 53 characters. Kubernetes 1.3 and earlier limited to only 24 characters (thus 14 character names).